spec/fizz_buzz_spec.rb

```diff
@@ -1 +1,60 @@
 require "fizz_buzz"
+
+describe FizzBuzz do
+  describe ".divisible_by" do
+    context "given empty string" do
+      it "returns zero" do
+        expect(FizzBuzz.divisible_by("")).to eql(0)
```

Comment:
I'd consider returning something other than zero here; maybe for now a string saying "not a number"